action/action_manager.go

```diff
@@ -0,0 +1,143 @@
+package action
```

Comment:
Can these be under `parade/`?  It's a utility library that I think @tzahij already asked for, so it seems reasonable to live there.

Reply:
I considered both options, 
I agree, changing it.